org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -1216,4 +1217,9 @@ public boolean isQueryTimeout(SQLException ex) {
 	public ITablePartitionService getTablePartitionService() {
 		return new TablePartitionService();
 	}
+
+	@Override
+	public String TO_Blob(String hexString) {

```

Comment:
I'm not sure if TO_Blob is the best name for this method

@hengsin ?

Also, I think this method requires some sanitization, can be misused if somebody passes a string that is not hexadecimal (f.e. containing quotes)

Reply:
I just follow the existing convention like the TO_CHAR method. Probably what we currently have in to_char or to_string. Alternative, is to pass in byte[] instead and perform the conversion of byte[] to hexString inside this method instead.

Reply:
OK, understood about the TO_Blob, fine by me.

As you suggest, it sounds safer to receive the byte[] as parameter and do the conversion inside.

Reply:
hi @uthadehikaru , as per the discussion above, please change the parameter to byte[] and perform the hex string conversion inside this method.